### ISSUE TITLE
Other module imports throw SerialException

### ISSUE THREAD

**Issue by Tyroo** [pyserial/pyserial#573]
![QQ图片20210407200132](https://user-images.githubusercontent.com/62151691/113864344-5edc6200-97dd-11eb-82c3-b588d44cca4d.png)
![QQ图片20210407200207](https://user-images.githubusercontent.com/62151691/113864368-656ad980-97dd-11eb-9112-9b37ae98373f.png)
![QQ图片20210407200437](https://user-images.githubusercontent.com/62151691/113864384-6b60ba80-97dd-11eb-90e5-c6a6a0bfafaa.png)
![QQ图片20210407200642](https://user-images.githubusercontent.com/62151691/113864397-70be0500-97dd-11eb-8262-8a9719becdaa.png)

**Comment by Tyroo** [comment]
Can sending and receiving only run in the same file?

**Comment by itchenfei** [comment]
com port can only open once， you import send.py in receive.py，so，when you run receive.py，send.py is already open com ports.

**Comment by NickCrews** [comment]
> com port can only open once， you import send.py in receive.py，so，when you run receive.py，send.py is already open com ports.

I don't think that is true. The port is only being opened once, either a) when you directly run `recieve.py` or b) when `recieve.py` is imported from `send.py`. I think this is actually a permissions issue with your port, it can't accept writes. Try just doing `serial.Serial(.....).write("aaa".encode("utf8"))` from the python terminal and see what you get?

**Comment by zsquareplusc** [comment]
you can't open the same port with two processes (on Windows) that's why there is an exception.

- for a loop-back connection (connected RX-TX) use the same `ser`object to send and receive within the same process
- alternatively connect two separate serial port with a crossed out cable ("nullmodem") but then open those ports separately in the two processes.
